src/js/index.js

```diff
@@ -113,6 +114,11 @@ class App {
         view: this.viewAdministrationIndex.bind(this),
         viewType: "Administration landing",
       },
+      {
+        path: new RegExp("^/my-municipality/notices/$"),
```

Comment:
why a dedicated route for this and not just use the class-based view approach for this?

Reply:
True, I reckon I can follow the same pattern I did for the `core.NoticePage`